### PR TITLE
add commit ignore for ESLint change

### PR DESCRIPTION
**Pull request by ghost**
### What
- 

### Why
- 

### Testing strategy
- [ ]

### CODE REVIEW

**Review by Ghost-in-a-Jar — APPROVED** [review]

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# ignores ESLint pr
```
Comment (pshapiro4broad) [inline .git-blame-ignore-revs]: Many git users have never used an ignore-revs file, so I think it'd be nice to include a comment that tells people how to enable it. For example: https://github.com/DataBiosphere/jade-data-repo/pull/1247/files

Comment (msilva-broad) [inline .git-blame-ignore-revs]: add Prettier to note, and a link to jira ticket?

Comment (nawatts) [reply .git-blame-ignore-revs]: Suggest adding this to https://github.com/DataBiosphere/terra-ui/wiki/Getting-started.

Comment (cahrens) [reply .git-blame-ignore-revs]: I tried running the `git config blame.ignoreRevsFile .git-blame-ignore-revs`, but when I annotate files in IntelliJ it's still showing the commit we are trying to ignore. I will start googling but am wondering if others already know what to do about this.

image

Comment (nawatts) [reply .git-blame-ignore-revs]: I wonder if IntelliJ is caching git config or something like that. `git blame` on the CLI seems to work ok.

File: .git-blame-ignore-revs [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# To configure git to use this file when using `git blame`:
+# $ git config blame.ignoreRevsFile .git-blame-ignore-revs
+
+# ignores ESLint pr: UIE-77: https://github.com/DataBiosphere/terra-ui/pull/3902
```
Comment (nawatts) [inline .git-blame-ignore-revs]: Reiterating @msilva-broad's [earlier comment](https://github.com/DataBiosphere/terra-ui/pull/3908#discussion_r1182868055): Prettier formatting, much more than anything ESLint related, is what we want to ignore from this commit.

**Review by msilva-broad — APPROVED** [review]
nice intro and item description

File: .git-blame-ignore-revs (hunk 1 above)
Comment (cahrens) [reply .git-blame-ignore-revs]: I restarted IntelliJ and it seems to be working now (only seeing the commit on lines that had never been touched before).

**Review by cahrens — APPROVED** [review]

**Review by pshapiro4broad — APPROVED** [review]
LGTM. Does this need a JIRA ticket? If so maybe it should use UIE-77?.

**Pull request merged**